opentaxii/utils.py

```diff
@@ -47,7 +47,6 @@ def initialize_api(api_config):
 
 
 def parse_basic_auth_token(token):
-    print("'{}'".format(token), len(token))
```

Comment:
ouch, that's a nasty one

Reply:
seriously

Reply:
@michaelknopf that's the truth

Reply:
I've made this change in master

Reply:
Thank you @traut.  Are there any plans to release a version including the fix?

Reply:
for sure. We just released 0.1.11 two weeks ago, so it might take some time to accumulate enough fixes / changes for next version though.